src/main/java/seedu/address/model/lesson/MakeUpLesson.java

```diff
@@ -15,26 +15,10 @@
      * @param subject Subject of the lesson.
      * @param homework Homework for the lesson.
      */
-    public MakeUpLesson(Date date, TimeRange timeRange,
-                           Subject subject, Set<Homework> homework) {
+    public MakeUpLesson(Date date, TimeRange timeRange, Subject subject, Set<Homework> homework) {
         super(date, timeRange, subject, homework);
     }
 
-    /**
```

Comment:
why need remove code when you refactor and format according to coding standards?